### PR TITLE
change default disk_type for GKE nodepool to null

### PR DESCRIPTION
**Pull request by chengcongdu**
GKE node-pool module currently have default disk_type as 'pd-standard' if not configured in blueprint, which is anti-pattern.
This PR change it to null and let GKE platform decide what default value should be.

Cloud build: https://console.cloud.google.com/cloud-build/builds/6ecb921a-0a98-44f7-ab43-bb22d2255f81?project=508417052821

*p.s. remember to run "make" after changing default value

### CODE REVIEW

File: tools/cloud-build/daily-tests/blueprints/ml-gke-e2e.yaml
Comment (ankitkinra) [inline tools/cloud-build/daily-tests/blueprints/ml-gke-e2e.yaml:1]: Should we remove the `disk_type` from one of the gpu pools to make sure that also is correctly identified ?

Comment (chengcongdu) [reply tools/cloud-build/daily-tests/blueprints/ml-gke-e2e.yaml:1]: the default_disk itself was tested in the added test. Are you more referring to the potential impact on ML functionality?

Comment (chengcongdu) [reply tools/cloud-build/daily-tests/blueprints/ml-gke-e2e.yaml:1]: updated

Comment (ankitkinra) [reply tools/cloud-build/daily-tests/blueprints/ml-gke-e2e.yaml:1]: Thanks, approved , please wait for the e2e test to pass (triggered them) before merging.

**Review by ankitkinra — APPROVED** [review]

**Pull request merged**